provisioning/templates/drupalvm-local.aliases.drushrc.php.j2

```diff
@@ -6,10 +6,26 @@
  * @see example.aliases.drushrc.php.
  */
 
-{% for vhost in apache_vhosts %}
-$aliases['{{ vhost.servername }}'] = array(
-  'uri' => '{{ vhost.servername }}',
-  'root' => '{{ vhost.documentroot }}',
+{% set devtool_hosts = [adminer_install_dir, pimpmylog_install_dir, '/usr/share/php/xhprof_html'] %}
```

Comment:
One note; I think the xhprof path differs on CentOS. I think we might need to add a variable upstream for that, similar to the other install_dir vars you're using.